tsconfig.json

```diff
@@ -12,7 +12,7 @@
     "removeComments": false,
     "sourceMap": false,
     "strict": true,
-    "target": "es2017",
+    "target": "es2022",
```

Comment:
WE can safely put a recent target as the consuming app will, in the end, transpile again using babel (or other tool)